Swift/ReplicatorConfiguration.swift

```diff
@@ -30,6 +30,17 @@ public enum ReplicatorType: UInt8 {
     case pull
 }
 
+/// Network Interface Experimental type
+///
+/// - useGetAddrInfo: use the POSIX function getAddrInfo to specify the network interface.
+/// - useBindFunction: use bind() function to bind socket to the specified network interface.
+/// - useNetworkFramework: use network framework to specify the network interface.
+public enum NetworkInterfaceExperimentalType: UInt8 {
+    case useGetAddrInfo = 0
```

Comment:
This shouldn't be in an experimental type. The another experiment will be using DNSService.